scenes/tileset.tres

```diff
@@ -8,11 +8,17 @@
 [ext_resource type="Texture2D" uid="uid://cnmmh3uq7rkla" path="res://assets/third_party/tiny-swords/Terrain/Bridge/Bridge_All.png" id="4_qml6a"]
 [ext_resource type="Texture2D" uid="uid://c7oht7wudd8wa" path="res://assets/first_party/tiles/Cliff_Tiles.png" id="5_jmclm"]
 [ext_resource type="Texture2D" uid="uid://brnsfo2pejt7i" path="res://assets/first_party/tiles/Void_Tiles_Stars_Level4.png" id="5_qpeek"]
+[ext_resource type="Texture2D" uid="uid://b8wlr1m0dj6xc" path="res://scenes/quests/story_quests/pixel/imagenes/Luna.png" id="6_em2qm"]
 [ext_resource type="Texture2D" uid="uid://pkm6smjl8861" path="res://assets/first_party/tiles/Grass_Tiles_Light.png" id="6_yc5yt"]
 [ext_resource type="Texture2D" uid="uid://dcq6v7mx3h1fl" path="res://assets/first_party/tiles/Dirt_Tiles.png" id="7_1srhd"]
+[ext_resource type="Texture2D" uid="uid://dx3gsy852or40" path="res://scenes/quests/story_quests/pixel/imagenes/ciudad.png" id="7_qwkka"]
 [ext_resource type="Texture2D" uid="uid://cledckq31klvq" path="res://assets/first_party/tiles/Cliff_Mines_Tiles.png" id="8_l50s2"]
+[ext_resource type="Texture2D" uid="uid://c4ehl08ag85n2" path="res://scenes/quests/story_quests/pixel/imagenes/pisoTierra.png" id="9_8nkvp"]
 [ext_resource type="Texture2D" uid="uid://r7scukca5yro" path="res://assets/first_party/tiles/Void_Tiles_Empty.png" id="9_hqjx3"]
 [ext_resource type="Texture2D" uid="uid://cn2j018vohdap" path="res://assets/first_party/tiles/Void_Tiles_Stars_Level1.png" id="10_kbenk"]
+[ext_resource type="Texture2D" uid="uid://c3oodkoeifcag" path="res://scenes/quests/story_quests/pixel/imagenes/plantas.png" id="10_rtrav"]
+[ext_resource type="Texture2D" uid="uid://0frletg7wjbm" path="res://scenes/quests/story_quests/pixel/imagenes/bigPlant.png" id="11_n0usd"]
+[ext_resource type="Texture2D" uid="uid://d4gfb4d12nbqf" path="res://scenes/quests/story_quests/pixel/imagenes/telefonoRojosTres.png" id="13_pfv7v"]
```

Comment:
Please do not modify the project-wide TileSet. If you want alternative tiles in your StoryQuest, please create a new TileSet in your StoryQuest directory.